logic.py

```diff
@@ -21,13 +21,23 @@ def execute(bot):
 #TODO: Implement these bot methods to bot
 
 class Forward(Order):
-    def execute(bot,distance):
-        bot.moveForward(distance)
+    def execute(bot):
+        bot.moveForward(1)
+        raise NotImplementedError()
+
+class ForwardTwice(Order):
+    def execute(bot):
+        bot.moveForward(2)
+        raise NotImplementedError()
```

Comment:
You implemented this and didn't implement this. Make up your mind! Don't raise `NotImplementedError`. Raise it in `move_forward` and `rotate`.